.cicdtemplate/.codemagic/codemagic.yaml

```diff
@@ -78,13 +83,24 @@ workflows:
       events:
         - push
       branch_patterns:
-        - pattern: main
+        - pattern: 'main'
+    environment:
+      vars:
+        # Increase the clone depth to 50 to cover all commits from the latest tag
+        CM_CLONE_DEPTH: 50
+      groups:
+        - firebase_credentials
+      java: 17
     scripts:
       - *detekt
       - *unit_test
       - name: Build APK for production
         script: |
           ./gradlew assembleProductionDebug -PversionCode=$BUILD_NUMBER
+      - name: Generating release notes with git commits from the latest tag
```

Comment:
@lydiasama forgot to update in other files. 😆

Reply:
Indeed. Thankss @AVI5HEK